packages/switch/src/component.tsx

```diff
@@ -17,16 +43,14 @@ export function Switch({ id, value, onClick, 'aria-label': ariaLabel, 'aria-labe
         aria-label={ariaLabel}
         aria-labelledby={ariaLabelledBy}
         aria-checked={value}
-        onClick={onClick}
-        className={classNames([ccSwitch.label, switchFocus])}
+        onClick={handleClick}
+        className={baseClasses}
+        tabIndex={disabled ? -1 : 0}
```

Comment:
I don't think this is needed if you have `disabled=true` on a button.

Reply:
Updated in this commit: [fb41ad8](https://github.com/warp-ds/react/pull/270/commits/fb41ad85ab2fbdb3561e8692df4e89c07f63fcb0)